lib/mayaUsd/ufe/private/Utils.cpp

```diff
@@ -21,13 +21,37 @@
 #include <pxr/usd/usd/primCompositionQuery.h>
 #include <pxr/usd/usdGeom/xformable.h>
 
+#include <maya/MGlobal.h>
 #include <ufe/log.h>
 
 #include <memory>
 #include <string>
 
 PXR_NAMESPACE_USING_DIRECTIVE
 
+namespace {
+// find position index for a given layer in the local LayerStack ( strong-to-weak order )
+uint32_t findLayerIndex(const PXR_NS::UsdAttribute& attr, const PXR_NS::SdfLayerHandle& layer)
+{
+    const auto& prim = attr.GetPrim();
+    const auto& stage = prim.GetStage();
+    const auto& layerStack = stage->GetLayerStack();
+
+    uint32_t position { 0 };
+
+    auto iter = std::find_if(
+        std::begin(layerStack), std::end(layerStack), [&](const PXR_NS::SdfLayerHandle& l) -> bool {
+            return layer == l;
+        });
+
+    if (iter != layerStack.end()) {
```

Comment:
Bit awkward that if iter == layerStack.end() we'll return position == 0.  Should we use TF_AXIOM(iter != layerStack.end())?